tests/garage/torch/algos/test_ppo.py

```diff
@@ -0,0 +1,51 @@
+"""
+This script creates a test that fails when garage.tf.algos.PPO performance is
+too low.
+"""
+import gym
+import torch
+
+from garage.envs import normalize
+from garage.envs.base import GarageEnv
+from garage.experiment import deterministic, LocalRunner
+from garage.np.baselines import LinearFeatureBaseline
+from garage.torch.algos import PPO
+from garage.torch.policies import GaussianMLPPolicy
+from tests.fixtures import snapshot_config
+
+
+class TestPPO:
+
+    @classmethod
```

Comment:
this should not be a class method.

if you want to do class-wide setup (shared by all test methods in the class), use `setup_class(cls)`/`teardown_class(cls)`

Reply:
delete `@classmethod` annotation